source/Misc.hpp

```diff
@@ -574,6 +563,20 @@ typedef enum
  * @}
  */
 
+/**
+ * @brief Solves a 6x6 system of equations M * a = b
+ *
+ * Uses Eigen::ColPivHouseholderQR which means it doesn't
+ * have any particular constraints on the matrix properties.
+ * Has high accuracy and good speed.
+ * 
+ * @param mat 6x6 Matrix (M in M * a = b)
+ * @param vec 6x1 Vector (b in M * a = b)
+ * @return vec6 Resulting solution (a in M * a = b)
+ */
+vec6
+solveMat6(const mat6& mat, const vec6& vec);
```

Comment:
I suppose here we can create a template function called solveLinSys or something like, with 2 specializations for mat & vec and for mat6 & vec6. That way it will look cleaner, don't you think.

I am not stopping the merge for that anyway

Reply:
We'll do this on the future